# Generated by Django 4.2.4 on 2023-12-29 18:20

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('content', '0004_alter_codeverification_created_at_and_more'),
    ]

    operations = [
        migrations.AddField(
            model_name='serie',
            name='students_complete_content',
            field=models.ManyToManyField(related_name='completed_series', to='content.student'),
        ),
        migrations.AddField(
            model_name='video',
            name='students_complete_content',
            field=models.ManyToManyField(related_name='completed_videos', to='content.student'),
        ),
        migrations.AlterField(
            model_name='codeverification',
            name='created_at',
            field=models.DateTimeField(default=datetime.datetime(2023, 12, 29, 18, 20, 3, 121472, tzinfo=datetime.timezone.utc), editable=False),
        ),
        migrations.AlterField(
            model_name='getoffer',
            name='start_date',
            field=models.DateField(default=datetime.datetime(2023, 12, 29, 18, 20, 3, 121472, tzinfo=datetime.timezone.utc), error_messages={'invalid': 'Invalid date format. Please provide a valid date.'}),
        ),
    ]
